### ISSUE TITLE
Accordion Block: Make accordion into a group.

### ISSUE THREAD

**Issue by joedolson** [WordPress/gutenberg#71780]
## What problem does this address?

The accordion group wrapper should be possible to name and handle as a group using `role="group"`. This allows users with screen readers to get an overall context for the accordion and know how large it is and their position within it.

## What is your proposed solution?

In the Advanced settings for the accordion, enable the option to set an accessible name for the group. When set, also add `role="group"` to the wrapper.

**Comment by t-hamano** [comment]
Thanks for the feedback!

> In the Advanced settings for the accordion, enable the option to set an accessible name for the group.

Does this mean that the `aria-label` attribute should be editable in the text field?

Also, does this block need to have a default label?

**Comment by joedolson** [comment]
Technically, I mean "assign an accessible name". The way the editor has mostly set that up is via `aria-label` attributes, but assigning a related heading using `aria-labelledby` would also be possible, and is technically a better solution - but is more difficult for users to implement. 

No, the block doesn't have to have a default label. A group isn't a landmark, so it can be unnamed.
